stumpy/mmotifs.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+import logging
+
+from .aamp_mmotifs import aamp_mmotifs
+from . import core, config, mdl, match, subspace
+
+logger = logging.getLogger(__name__)
+
+
+@core.non_normalized(aamp_mmotifs)
+def mmotifs(
+    T,
+    P,
+    I,
+    min_neighbors=1,
+    max_distance=None,
+    cutoffs=None,
+    max_matches=10,
+    max_motifs=1,
+    k_s=None,

```

Comment:
Can this just be `k` instead of `k_s`? 

Can we move `k` and `include ` to after the `atol` parameter?

Reply:
> Can this just be k instead of k_s?

I also wanted to do it like this. But I think that it's only working for the first motif, if the user cooses `k=None` since `k` has another value after finding the first motif. Or have I missed something?

Reply:
I see what you mean. I think that at the top of the function, you can add a:

```
T = core._preprocess(T)
m = T.shape[-1] - P.shape[-1] + 1
reset_k = False
```

and then later do something like:

```
mdls, subspaces = mdl(T, m, candidate_idx, nn_idx, include)
if k is None:
    k = np.argmin(mdls)
    reset_k = True
subspace_k = subspaces[k]
``` 

and then at the end of the while loop, you'd add:

```
candidate_idx = np.argmin(P, axis=1)
nn_idx = I[np.arange(len(candidate_idx)), candidate_idx]
if reset_k:
    k = None
```